kafka/4.4/modules/ROOT/pages/kafka-connector-reference.adoc

```diff
@@ -0,0 +1,1999 @@
+| Default record limit a| Number |  The maximum number of records returned on a poll call to the Kafka cluster. You can override this parameter at the source level. |  `500` |
+| DNS Lookups a| Enumeration, one of:
+
+** DEFAULT
+** USE_ALL_DNS_IPS
+** RESOLVE_CANONICAL_BOOTSTRAP_SERVERS_ONLY |  Controls how the client uses DNS lookups:
+
+* If set to `use_all_dns_ips`, when the lookup returns multiple IP addresses for a hostname, a connection is attempted to all of the IP addresses before the connection fails. This value applies to both bootstrap and advertised servers.
+* If set to `resolve_canonical_bootstrap_servers_only`, each entry is resolved and expanded into a list of canonical names. | `DEFAULT` |
+| Heartbeat interval a| Number |  The expected time between heartbeats to the consumer coordinator when using Apache Kafka's group management facilities. Heartbeats are used to ensure that the consumer's session stays active and to facilitate rebalancing when new consumers join or leave the group. The value must be set lower than the `Session timeout` parameter, but typically should be set no higher than 1/3 of that value. You can set this value lower to control the expected time for normal rebalances. | `3` |
+| Heartbeat Interval Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  The time unit for fetching the heartbeat interval time scalar. |  `SECONDS` |
+| Session Timeout a| Number |  The timeout used to detect consumer failures when using Apache Kafka's group management facility. The consumer sends periodic heartbeats to indicate its aliveness to the broker. If no heartbeats are received by the broker before the expiration of this session timeout, the broker removes this consumer from the group and initiates a rebalance. The value must be in the allowable range as configured in the broker configuration by `group.min.session.timeout.ms` and `group.max.session.timeout.ms`. |  `10` |
+| Session timeout time unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  The time unit for session timeout scalar |  `SECONDS` |
+| Connection maximum idle time a| Number |  Closes idle connections after the number of milliseconds specified by this configuration |  `540` |
+| Connection maximum idle time time unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  The time unit for the connection's maximum idle time scalar. |  `SECONDS` |
+| TLS Configuration a| <<Tls>> | Defines a TLS configuration, which can be used from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol, you must configure at least the keystore in the `tls:context` child element of the `listener-config`. | |
```

Comment:
Just verify we are using the correct description for this field in all occurrences.